readme.md

```diff
@@ -1,40 +1,37 @@
-<!-- trunk-ignore(markdownlint/MD041) -->
-<p align="center">
-  <a href="https://docs.trunk.io">
-    <img src="https://static.trunk.io/assets/trunk_plugins_logo.png" />
-  </a>
-</p>
-<p align="center">
-  <a href="https://marketplace.visualstudio.com/items?itemName=Trunk.io">
-    <img src="https://img.shields.io/visual-studio-marketplace/i/Trunk.io?logo=visualstudiocode"/>
-  </a>
-  <a href="https://slack.trunk.io">
-    <img src="https://img.shields.io/badge/slack-slack.trunk.io-blue?logo=slack"/>
-  </a>
-  <a href="https://docs.trunk.io">
-    <img src="https://img.shields.io/badge/docs.trunk.io-7f7fcc?label=docs&logo=readthedocs&labelColor=555555&logoColor=ffffff"/>
-  </a>
-  <a href="https://trunk.io">
-    <img src="https://img.shields.io/badge/trunk.io-enabled-brightgreen?logo=data:image/svg%2bxml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGZpbGw9Im5vbmUiIHN0cm9rZT0iI0ZGRiIgc3Ryb2tlLXdpZHRoPSIxMSIgdmlld0JveD0iMCAwIDEwMSAxMDEiPjxwYXRoIGQ9Ik01MC41IDk1LjVhNDUgNDUgMCAxIDAtNDUtNDVtNDUtMzBhMzAgMzAgMCAwIDAtMzAgMzBtNDUgMGExNSAxNSAwIDAgMC0zMCAwIi8+PC9zdmc+"/>
-  </a>
-  </a>
-  <a href="https://api.securityscorecards.dev/projects/github.com/trunk-io/plugins">
-    <img src="https://api.securityscorecards.dev/projects/github.com/trunk-io/plugins/badge"/>
-  </a>
-  <a href="https://bestpractices.coreinfrastructure.org/projects/7006">
-    <img src="https://bestpractices.coreinfrastructure.org/projects/7006/badge">
-  </a>
-</p>
+<!-- markdownlint-disable first-line-heading -->
+
+[![Trunk.io](https://static.trunk.io/assets/trunk_plugins_logo.png)](https://trunk.io)
+
+[![docs](https://img.shields.io/badge/-docs-darkgreen?logo=readthedocs&logoColor=ffffff)][docs]
+[![slack](https://img.shields.io/badge/-slack-611f69?logo=slack)][slack]
+[![vscode](https://img.shields.io/visual-studio-marketplace/i/trunk.io?color=0078d7&label=vscode&logo=visualstudiocode)][vscode]
+[![openssf](https://api.securityscorecards.dev/projects/github.com/trunk-io/trunk-action/badge)](https://api.securityscorecards.dev/projects/github.com/trunk-io/trunk-action)
```

Comment:
This is the wrong badge link. Should be plugins, not action

Reply:
Wow nice catch, I never would have noticed that 🥇